rpc/send.go

```diff
@@ -125,28 +124,18 @@ func Send(opts Options, method string, addrs []net.Addr, getArgs func(addr net.A
 		}
 	}
 
-	var clients []*Client
+	if opts.Timeout == 0 {
+		// Wait forever.
+		opts.Timeout = math.MaxInt64
+	}
+
+	var orderedAddrs []net.Addr
 	switch opts.Ordering {
 	case OrderStable:
-		for _, addr := range addrs {
-			clients = append(clients, NewClient(addr, nil, context))
-		}
+		orderedAddrs = addrs
 	case OrderRandom:
-		// Randomly permute order, but keep known-unhealthy clients last.
```

Comment:
Why did you remove the "healthy first" optimization? I'm not seeing anything that does an equivalent job.